documentation/modules/cruise-control/proc-fixing-problems-with-kafkarebalance.adoc

```diff
@@ -26,13 +26,20 @@ During a “refresh”, a new optimization proposal is requested from the Cruise
 
 .Procedure
 
-. Get information about the error from the `KafkaRebalance` status and resolve the issue if possible.
+. Get information about the error from the `KafkaRebalance` status:
++
+[source,shell,subs="+quotes"]
+----
+kubectl get kafka kafkarebalance -o jsonpath='{.status}'
```

Comment:
I thought we has switched to using `describe`? We should be consistent with whichever command was used elsewhere?

Reply:
Well spotted. I changed two more "check status" commands to use `kubectl describe`.